tests/config/ncsa/ForwarderCfg.yaml

```diff
@@ -0,0 +1,57 @@
+# name of forwarder
+NAME: f99
```

Comment:
Consider naming this differently;  if we have multiple forwarders (as a backup, in case one fails), it might be better to either use the system's name, or the system's name and the process id.  Otherwise we'll have a bunch of config files across multiple machines with only "NAME" being different in all the configs.

Reply:
I have created a [ticket](https://jira.lsstcorp.org/browse/DM-23819) to fix this.